node/src/command.rs

```diff
@@ -326,6 +326,7 @@ pub fn run() -> Result<()> {
 		// Fix as this run the node
 		None => {
 			let runner = cli.create_runner(&cli.run.normalize())?;
+			let collator_options = cli.run.collator_options();
```

Comment:
i'm assuming there are defaults that come out of this function? cause it doesn't seem like we're adding any specific options here

Reply:
`	/// Create [`CollatorOptions`] representing options only relevant to parachain collator nodes
	pub fn collator_options(&self) -> CollatorOptions {
		CollatorOptions { relay_chain_rpc_url: self.relay_chain_rpc_url.clone() }
	}`